### PR TITLE
Improve Typescript typings

### PR DESCRIPTION
**Pull request by fabiendem**
Improve the Typescript typings.
Didn't add methods which are not in both iOS and Android yet.

Interesting bits:

Expose `scaleType` (`type scaleType = "fit" | "fill";`) on `<TwilioVideoParticipantView>` so we can do:
```jsx
<TwilioVideoParticipantView
	...
	scaleType={"fit"}
/>
```

Expose `getStats ` & `onStatsReceived`.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,18 +1,22 @@
-declare module "react-native-twilio-video-webrtc"{
+declare module "react-native-twilio-video-webrtc" {
   import { ViewProps } from "react-native";
   import React from "react";
 
-  interface TwilioVideoLocalViewProps extends ViewProps {
-    enabled: boolean;
-    ref?: React.Ref<any>;
```
Comment (blindgaenger) [inline index.d.ts:7]: Was the `ref` removed intentionally?

**Review by blindgaenger — COMMENTED** [review]
That's awesome. I've just started to patch my local types to check `getStats` out. Glad you took the effort! 🙇‍♂️

**Comment by blindgaenger** [comment]
I think the data track callbacks are also implemented on both platforms:

```
onDataTrackMessageReceived
onParticipantAddedDataTrack
onParticipantRemovedDataTrack
```

File: index.d.ts (hunk 1 above)
Comment (fabiendem) [reply index.d.ts:7]: Nope my bad I moved the interfaces around.
It's there line 15/13 https://github.com/blackuy/react-native-twilio-video-webrtc/pull/407/files#diff-7aa4473ede4abd9ec099e87fec67fd57afafaf39e05d493ab4533acc38547eb8R13

File: index.d.ts [hunk 2]
```diff
@@ -72,6 +76,7 @@ declare module "react-native-twilio-video-webrtc"{
     onRoomDidFailToConnect?: RoomErrorEventCb;
     onRoomParticipantDidConnect?: ParticipantEventCb;
     onRoomParticipantDidDisconnect?: ParticipantEventCb;
+    onStatsReceived?: any;
```
[the remark this answers is not in the document]
Comment (fabiendem) [reply index.d.ts]: Nice, I went too fast here.

**Comment by fabiendem** [comment]
> I think the data track callbacks are also implemented on both platforms

Indeed, failed the 7 differences game, will add them now

**Comment by fabiendem** [comment]
> I think the data track callbacks are also implemented on both platforms

Added 👍

**Review by blindgaenger — APPROVED** [review]
Awesome, that's it!

File: index.d.ts (hunk 1 above)
Comment (blindgaenger) [reply index.d.ts:7]: Sorry to get back to this, but `TwilioVideoLocalViewProps` is missing the `ref`. The `TwilioVideoParticipantViewProps` still have it, though.

Comment (fabiendem) [reply index.d.ts:7]: That's my bad again, fixed.
Thanks for flagging this

**Review by slycoder — APPROVED** [review]
Thank you @fabiendem  for this change and @blindgaenger for the thorough review!

**Pull request merged**